### PR TITLE
fix issue with splunk

### PR DESCRIPTION
**Pull request by alexjoybc**
# Description

This PR includes the following proposed change(s):

- fix splunk configuration for production env

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Running the app locally and on openshift

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/dev-efiling-reviewer-api-build.yaml [hunk 1]
```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [master]
   workflow_dispatch:
-    branches: [master]
```
Comment (akroon3r) [inline .github/workflows/dev-efiling-reviewer-api-build.yaml:7]: if you're removing this, remove the event too please

**Review by akroon3r — COMMENTED** [review]
One comment

File: .github/workflows/dev-efiling-reviewer-api-build.yaml (hunk 1 above)
Comment (alexjoybc) [reply .github/workflows/dev-efiling-reviewer-api-build.yaml:7]: ahahaha

Comment (alexjoybc) [reply .github/workflows/dev-efiling-reviewer-api-build.yaml:7]: I was pushing from my branch

File: .github/workflows/dev-efiling-reviewer-api-build.yaml [hunk 2]
```diff
@@ -4,7 +4,6 @@ on:
   push:
     branches: [master]
   workflow_dispatch:
```
Comment (akroon3r) [inline .github/workflows/dev-efiling-reviewer-api-build.yaml:6]:
```suggestion
```

Comment (akroon3r) [inline .github/workflows/dev-efiling-reviewer-api-build.yaml:6]:
```suggestion
  workflow_dispatch:
    branches: [master]
```

**Review by akroon3r — APPROVED** [review]

**Pull request merged**